phoenix-core/src/main/java/org/apache/phoenix/index/GlobalIndexChecker.java

```diff
@@ -237,6 +241,7 @@ private void repairIndexRows(byte[] indexRowKey, long ts, List<Cell> row) throws
             }
             // Rebuild the index rows from the corresponding the rows in the the data table
             byte[] dataRowKey = indexMaintainer.buildDataRowKey(new ImmutableBytesWritable(indexRowKey), viewConstants);
+            LOG.debug("repairIndexRows : " + Bytes.toStringBinary(dataRowKey) + " --> " + Bytes.toStringBinary(indexRowKey));
```

Comment:
Log should be at TRACE, both because it could be quite verbose, and because some operators have restrictions about logging customer data, which can be contained in row keys.

Reply:
I will remove the log. It was not really important to have it. We can have metrics instead.

Reply:
Done